client/app/states/vms/details/details.html

```diff
@@ -0,0 +1,513 @@
+<ol class="breadcrumb">
+  <li>
+    <a ui-sref="services">{{'My Services'|translate}}</a>
+  </li>
+  <li>
+    <a ui-sref="services.details({serviceId: vm.vmDetails.service.id})">{{ ::vm.vmDetails.service.name }}</a>
+  </li>
+  <li>
+    <a ui-sref="vms.details({vmId: vm.vmDetails.id})">{{ ::vm.vmDetails.name }}</a>
```

Comment:
YAY SINGLE BINDING! any reason why the rest of this isn't? are we polling the collection?  (if not, and there isn't a chance it'll be updated, all in this should be single bound, colon colon **FOREVER**)